Essentials/src/main/resources/config.yml

```diff
@@ -454,11 +454,20 @@ remove-god-on-disconnect: false
 # Set to -1 for no timeout.
 auto-afk: 300
 
-# Auto-AFK Kick
-# After this timeout in seconds, the user will be kicked from the server.
-# essentials.afk.kickexempt node overrides this feature.
+# Auto-AFK Actions Time
+# After this timeout in seconds, the actions defined in auto-afk-actions will be executed.
+# essentials.afk.actionsexempt node overrides this feature.
 # Set to -1 for no timeout.
-auto-afk-kick: -1
+auto-afk-actions-timeout: -1
+
+# Auto-AFK Actions
+# The actions to execute after the auto-afk-actions-timeout.
+# Available placeholders:
+# {PLAYER} - The player's displayname.
+# {USERNAME} - The player's username.
+# {WORLDNAME} - The full name of the player's current world.
+auto-afk-actions:
+  - "kick {USERNAME}"
```

Comment:
This change means players will no longer see the configured AFK kick message. AFK kicks should be kept separate from other AFK timeout actions to avoid breaking existing setups.

I'm also not convinced that running all commands at once is a good design for this feature. It would be far more useful to allow setting different timeouts for each action.